sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -6,15 +6,15 @@
 
 from enum import Enum
 from typing import TYPE_CHECKING
-from ._generated.models import ContentProperties
+from ._generated.models import ContentProperties as GeneratedContentProperties
 
 if TYPE_CHECKING:
     from ._generated.models import ManifestAttributesBase
     from ._generated.models import RepositoryProperties as GeneratedRepositoryProperties
     from ._generated.models import ArtifactTagProperties as GeneratedTagProperties
 
 
-class ContentPermissions(object):
+class ContentProperties(object):
```

Comment:
Is this model used as input? Might want to document the keywords that can be passed, if so